docs/notebooks/credit_card_dp_notebook.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# differentially private synthetic model with credit card dataset.\n",
+    "\n",
+    "This blueprint implements a practical attack on various parameters and privacy settings of a synthetic model to measure its ability in momerizing the canaries inserted into the training set.\n",
```

Comment:
spelling correction - memorizing

Reply:
Also recommend saying 
`This blueprint implements a practical attack on a credit card dataset. We tune various parameters and privacy settings of a synthetic model to measure its ability in memorizing canaries inserted into dataset. We show that enabling differential privacy (DP) can provide greater protection from memorization of canaries.